data/presets/waterway/access_point.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
probably icon distinct from one used for unrecognised tags would make sense, but I have no idea for anything

Reply:
maybe the coastline icon would fit?

<img width="82" height="66" alt="image" src="https://github.com/user-attachments/assets/fa2deac1-c7fe-4da4-b851-d0a1c61e4893" />